cmd/object-handlers.go

```diff
@@ -2830,7 +2830,6 @@ func (api objectAPIHandlers) DeleteObjectHandler(w http.ResponseWriter, r *http.
 
 	rcfg, _ := globalBucketObjectLockSys.Get(bucket)
 	if rcfg.LockEnabled && opts.DeletePrefix {
-		writeErrorResponse(ctx, w, toAPIError(ctx, errors.New("force-delete is forbidden in a locked-enabled bucket")), r.URL)
```

Comment:
We have to return this error however we have to convert it into something more meaningful so that the logging doesn't get triggered.

First, do this

```
apiErr := toAPIError(ctx, errInvalidArgument)
apiErr.Description = "force-delete is forbidden on Object Locking enabled buckets" 
writeErrorResponse(ctx, w, apiErr, r.URL)
return
```